Makefile

```diff
@@ -145,6 +145,13 @@ scoop: all
 	@ echo "Running minimal example with SCOOP"
 	python -m scoop -n 2 rmg.py -v testing/scoop/input.py
 
+update: all
+	@ echo "Updating RMG-py"
+	git stash || true
+	git pull git@github.com:ReactionMechanismGenerator/RMG-Py.git main || git pull https://github.com/ReactionMechanismGenerator/RMG-Py.git main || true
+	make
```

Comment:
Do we assume the Fonda env is activated?

Reply:
It has to be active if the user used `make` to activate the `make update`